src/plugins/chart_expressions/expression_pie/public/components/chart_split.tsx

```diff
@@ -45,7 +35,7 @@ export const ChartSplit = ({
           }
           return spec.id;
         }}
-        sort={sort}
+        sort="dataIndex"
```

Comment:
It make sense to me, @dej611  could you please confirm that it's ok

Reply:
Seems fine. But I could not find any documentation about the sorting predicate in the charting library. Maybe @markov00 or @nickofthyme could clarify its usage.

Reply:
the `dataIndex` predicate means: don't sort, take as it comes from the dataset, FIFO.
Please @alexwizp use the enum `Predicate.DataIndex` rather then a string there

Reply:
Ok.... in that case it exactly what we need. Thank you all